lib/usd/translators/meshWriter.cpp

```diff
@@ -129,21 +115,22 @@ PxrUsdTranslators_MeshWriter::writeMeshAttrs(
     // determine whether we use the "input" or "final" mesh when exporting
     // mesh geometry. This should only be run once at default time.
     if (usdTime.IsDefault()) {
+
         const TfToken& exportSkin = _GetExportArgs().exportSkin;
         if (exportSkin != UsdMayaJobExportArgsTokens->auto_ &&
             exportSkin != UsdMayaJobExportArgsTokens->explicit_) {
-        
-            _skelInputMesh = MObject();
-        
+
+            m_skelInputMesh = MObject();
+
         }else if (exportSkin == UsdMayaJobExportArgsTokens->explicit_ &&
                   !UsdSkelRoot::Find(primSchema.GetPrim())) {
-        
-            _skelInputMesh = MObject();
-        
+
+            m_skelInputMesh = MObject();
+
         }else {
 
             SdfPath skelPath;
-            _skelInputMesh = UsdMayaJointUtil::writeSkinningData(primSchema,
+            m_skelInputMesh = UsdMayaJointUtil::writeSkinningData(primSchema,
```

Comment:
More irrelevant format and naming changes here and below. Those should go in a dedicated commit to avoid clouding the important changes.

Reply:
I agree, probably too late now but I reverted m_skelInputMesh  to _skelInputMesh . Again these changes dates back a while before we decided on guidelines, etc...

See 35c0c0fd6964f2970a512397546036a73af201df